test/python/test_authenticators_api.py

```diff
@@ -0,0 +1,37 @@
+from __future__ import absolute_import
+
+import unittest
+
+import openapi_client
+
+from . import api_config
+
+AUTHENTICATOR_FIELDS = [
+    "enabled",
+    "installed",
+    "configured",
+]
+
+class TestAuthenticatorsApi(api_config.ConfiguredTest):
```

Comment:
Let's verify that a successful response contains the `authn` authenticator in the three different arrays, and lets verify that an unsuccessful response returns the 401 error.